homeassistant/components/roon/media_player.py

```diff
@@ -98,24 +98,19 @@ def __init__(self, server, player_data):
         """Initialize Roon device object."""
         self._remove_signal_status = None
         self._server = server
-        self._available = True
-        self._last_position_update = None
```

Comment:
Are you sure it's safe to delete this line?

Reply:
Replaced with `_attr_media_position_updated_at`. The default value is `None` in the base class.